src/modules/extendedCallWindow/docs/de_DE.md

```diff
@@ -1,30 +1,62 @@
+![Tabelle mit zusaätzlich benötigten Fahrzeugen](fahrzeugeVerschoenern.png)
+
+### Tooltips
+Mit dieser Einstellung werden zu den Buttons von "Benötigste Fahrzeuge verschönern" kleine Hinweisfelder ergänzt, die erscheinen, sobal man die Maus kurz auf dem jeweiligen Button ruhen lässt.
+
+### Maximalbesetzung
+Manchmal wird am Einsatzort ja auch eine bestimmte Personalmenge gebraucht. Dieses Feature ergänzt die Personalrechnung, indem es davon ausgeht, dass jedes fahrzeug mit der Maximalen Personalmenge ausrückt.
 
 ## Patientenzusammenfassung
+Dieses Modul ist perfekt für alle Spieler, die bei vielen Patienten gerne den Überblick behalten. Es zeigt, wie viele Rettungsfahrzeuge welcher Art noch am Einsatzort benötigt werden.
+![Patientenliste mit Zussamenfassung](patientList.png)
 
 ## AAO-Zähler
+Du willst wissen, wie oft bereits auf eine AAO geklickt wurde? Dann aktiviere dieses Modul, es zählt nämlich, wie oft auf eine AAO geklickt wurde. Natürlich gibt es auch einen Knopf zum Zurücksetzen des Zählers.
+![AAO-Zähler](aao2.png)
 
 ### AAO-Zähler als Badge
+Dieses Modul ist eine kleine Spielerei zum Modul AAO-Zähler. Seine Funktion ist, dass die Anzahl, wie oft auf eine AAO geklickt wurde, neben der AAO als kleiner roter Kreis angezeigt wird. Hierfür muss "AAO-Zähler" aktiviert sein.
+![AAO mit Fahrzeugzähler als Badge](aao3.png)
 
 ## angeklickte AAOs hervorheben
+Vielleicht kennst du das ja, du willst schnell wissen, welche AAO du schon angeklickt hast. Dieses Modul ist perfekt dafür, man sieht nämlich mit einem farbigen Rand, welche AAO man schon angegklickt hat.
+![Hervorgehobene AAO](aao1.png)
 
 ### Farbe der Hervorhebung
+Natürlich kannst du dir die Farbe, in der die AAO's hervorgehoben werden, selber einstellen. Nutze dazu den Color-Picker in den Einstellungen.
 
 ### Breite der Hervorhebung
+Natürlich kannst du dir auch die Breite der Hervorhebung anpassen. gemessen wird hier in Pixeln.
+
+Hier ein Bild, wie es mit einem lilanen Farbton (RGB: 230, 51, 197) und einem 5 Pixel breitem Rand aussieht:
+![Bild einer hervorgehobenen AAO mit einem lilanen Farbton (RGB: 230, 51, 197) und einem 5 Pixel breitem Rand](aao4.png)
 
 ## Fahrzeugauswahl zurücksetzen
+Neben dem Button `Hilfe zu diesem Einsatz` erscheint wie von zauberhand ein weiterer Button, der alle bereits ausgewählten Fahrzeuge zurücksetzt.
+![Zurücksetzen-Button](fahrzeugwahlReset.png)
 
-## unpassende AAOs ausgrauen
+## Unpassende AAOs ausgrauen
+Suchst du immer ewigkeiten die passende AAO, weil du für jeden Einsatz eine andere AAO hast? Dieses Modul wirst du lieben. Es garut alle AAO's, in denen nicht mindestens ein Teil des Einsatzstichwortes vorkommt, aus.
+![ausgegraute AAO's](aao5.png)
 
 ## AAO-Anfahrtszeit anzeigen
+Diese Einstellung zeigt an, wie lange das am längsten brauchende Fahrzeug der AAO braucht. Erscheint, die Maus kurz auf der AAO ruhen lässt.
+![AAO mit angezeigter Anfahrtszeit des am längsten brauchenden Fahrzeuges](aao6.png)
```

Comment:
Is this the LSSM Version or ingame Version?
Doesn't LSSM show the time in a green box?

Reply:
This ist the Version of the LSSM, its in the AAO direcly

Reply:
Oh yeah, I see.
Does not look that beatiful xD